assets/app/view/game_card.rb

```diff
@@ -132,8 +132,10 @@ def render_link(href, click, text)
       )
     end
 
-    def time_or_date(ts)
-      ts > Time.now - 82_800 ? ts.strftime('%T') : ts.strftime('%F')
+    def render_time_or_date(timestamp)
+      ts = Time.at(@gdata[timestamp].to_i)
```

Comment:
this will crash if there's no data should be safer here and do

@gdata[ts_key]&.to_i || 0